test/OpenTelemetry.Instrumentation.Dependencies.Tests/docker-compose.integration.yml

```diff
@@ -8,7 +8,8 @@ services:
     image: mcr.microsoft.com/mssql/server:2019-latest
     environment:
       - ACCEPT_EULA=Y
-      - SA_PASSWORD=Pass@word
+      # Note: This password is for the ephemeral sql instance running in the container used for tests. Nothing that needs to be handled securely.
```

Comment:
👍

Reply:
https://github.com/microsoft/ApplicationInsights-dotnet/blob/develop/WEB/Test/E2ETests/E2ETests/docker-compose452AppOn462.yml#L37-L42 what application insights have for similar tes.t